### PR TITLE
Croissants

### PR DESCRIPTION
**Pull request by kol1th**
Adds a croissant. Tested and sprited by myself :tada: 
As the changelog says, it's made with 5u water, 5u flour, 5u milk, and one egg in the microwave.

image

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -4060,3 +4060,14 @@
 			if(counter == available_snacks.len)
 				counter = 0
 			counter++
+
+
+/obj/item/weapon/reagent_containers/food/snacks/croissant
+	name = "croissant"
+	desc = "True french quisine."
```
Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Cuisine.

Comment (kol1th) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: god damnit

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4060,3 +4060,14 @@
...
+	icon_state = "croissant"
+
+	New()
```
Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Bad practice. Replace this with 
/obj/item/weapon/reagent_containers/food/snacks/croissant/New()

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -4060,3 +4060,14 @@
...
 				counter = 0
 			counter++
+
+
+/obj/item/weapon/reagent_containers/food/snacks/croissant
+	name = "croissant"
+	desc = "True french cuisine."
+	icon_state = "croissant"
+
+	New()
```
Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: bad practice, replace this with /obj/item/weapon/reagent_containers/food/snacks/croissant/New()

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -4060,3 +4060,13 @@
...
 				counter = 0
 			counter++
+
+
+/obj/item/weapon/reagent_containers/food/snacks/croissant
+	name = "croissant"
+	desc = "True french cuisine."
+	icon_state = "croissant"
+/obj/item/weapon/reagent_containers/food/snacks/croissant/New()
+		..()
```
Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4164]: GEE @kol1th HOW COME YOUR BYOND LETS YOU HAVE TWO INDENTS

Comment (kol1th) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4164]: on the `..()`?

Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4164]: just unident this whole proc by one

**Comment by kol1th** [comment]
Oh and needs wiki since I'm inept :ok_hand:

**Comment by ScrambledSteak** [comment]
Does this mean I can make chef gimmicks where I'm just Carl from Jimmy Neutron and ask people if they're gonna finish their croissants

**Comment by kol1th** [comment]
> conflicts 

???

**Comment by Shadowmech88** [comment]
Kurfurst's food PR was merged, it probably touched the same files that this touches.
Just rebase to fix the conflicts.

**Comment by kol1th** [comment]
did that work I don't really know what I did

**Comment by kol1th** [comment]
What the fuck is going on

**Comment by kol1th** [comment]
I'm just going to remove this and mess with it until I can fix it

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 5]
```diff
@@ -4060,3 +4060,108 @@
 			if(counter == available_snacks.len)
 				counter = 0
 			counter++
+
+<<<<<<< 1a4d699ddeeb2b2d33c10761ce0b4bd3b1588279
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: just need to remove this line

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 4 above)
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: also do what he said, remove one tab from this proc

**Comment by Intigracy** [comment]
You can't get rebasing to work?

**Comment by kol1th** [comment]
life is truly, full of mistakes

**Comment by kol1th** [comment]
And I keep pressing the wrong button. I can't figure it out no

**Pull request closed without merging**